cli/ballerina-cli/src/main/java/io/ballerina/cli/task/RunNativeImageTestTask.java

```diff
@@ -55,7 +55,6 @@
 
 import static io.ballerina.cli.launcher.LauncherUtils.createLauncherException;
 import static io.ballerina.cli.utils.NativeUtils.createReflectConfig;
-import static io.ballerina.cli.utils.NativeUtils.createResourceConfig;
```

Comment:
Do we use this `createResourceConfig` method anywhere else? 
If not we can remove this method from `io.ballerina.cli.utils.NativeUtils` right?

Reply:
The function mocking implementation will eventually make use of the createResourceConfig and the associated classes. So im leaving it in for now.